service/controller/key/testdata/case-1-unit-render.golden

```diff
@@ -0,0 +1,171 @@
+10-giantswarm-extra-args.conf: |-
+  [Service]
+  CPUAccounting=true
+  MemoryAccounting=true
+  Slice=kubereserved.slice
+  Environment="DOCKER_CGROUPS=--exec-opt native.cgroupdriver=cgroupfs --cgroup-parent=/kubereserved.slice --log-opt max-size=25m --log-opt max-file=2 --log-opt labels=io.kubernetes.container.hash,io.kubernetes.container.name,io.kubernetes.pod.name,io.kubernetes.pod.namespace,io.kubernetes.pod.uid"
+  Environment="DOCKER_OPT_BIP=--bip=
+audit-rules.service: |
+  [Service]
+  ExecStartPre=/bin/bash -c "while [ ! -f /etc/audit/rules.d/10-docker.rules ]; do echo 'Waiting for /etc/audit/rules.d/10-docker.rules to be written' && sleep 1; done"
+containerd.service: |
+  [Service]
+  CPUAccounting=true
+  MemoryAccounting=true
+  Slice=kubereserved.slice
+etcd3-defragmentation.service: |-
+  [Unit]
+  Description=etcd defragmentation job
+  After=docker.service etcd3.service
+  Requires=docker.service etcd3.service
+  [Service]
+  Type=oneshot
+  EnvironmentFile=/etc/network-environment
+  Environment=IMAGE=
```

Comment:
Is that empty on purpose or just WIP?

Reply:
WIP - I need to fix the template names inside the units.
Will do so in a followup.